### PR TITLE
Add local API authorization

### PR DESCRIPTION
**Pull request by tuohai666**

### CODE REVIEW

**Comment by AhahaGe** [comment]
please check ci error

**Comment by tuohai666** [comment]
> please check ci error

Thanks, I have fixed it.

File: shenyu-web/src/main/java/org/apache/shenyu/web/filter/LocalDispatcherFilter.java [hunk 1]
```diff
@@ -67,7 +66,7 @@ public LocalDispatcherFilter(final DispatcherHandler dispatcherHandler) {
         String path = exchange.getRequest().getURI().getPath();
         if (PathMatchUtils.match(DISPATCHER_PATH, path)) {
             String key = exchange.getRequest().getHeaders().getFirst(Constants.X_ACCESS_TOKEN);
-            if (Objects.isNull(sha512Key) || !ShaUtils.shaEncryption(sha512Key).equals(key)) {
+            if (Objects.isNull(sha512Key) || !sha512Key.equalsIgnoreCase(ShaUtils.shaEncryption(key))) {
```
Comment (yu199195) [inline shenyu-web/src/main/java/org/apache/shenyu/web/filter/LocalDispatcherFilter.java]: ||  Objects.isNull(key)

Comment (tuohai666) [reply shenyu-web/src/main/java/org/apache/shenyu/web/filter/LocalDispatcherFilter.java]: Fixed.

**Review by yu199195 — APPROVED** [review]

**Pull request merged**